docs/app/partials/layout-alignment.tmpl.html

```diff
@@ -4,47 +4,72 @@
 
     <p><code>layout="column" layout-align="center end"</code> would make
     children align along the center vertically and along the end (right) horizontally. </p>
+  
+  
   <table class="md-api-table">
+         <thead>
+           <tr>
+             <th>API</th>
+             <th>Sets child alignments within the layout container</th>
+           </tr>
+         </thead>
+          <tr>
+            <td>layout-align</td>
+            <td>Sets default alignment unless overridden by another breakpoint.</td>
+          </tr>
     <tr>
-      <td>layout-align</td>
-      <td>Sets child alignment.</td>
-    </tr>
-    <tr>
-      <td>layout-align-sm</td>
-      <td>Sets child alignment on devices less than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>layout-align-gt-sm</td>
-      <td>Sets child alignment on devices greater than 600px wide.</td>
-    </tr>
-    <tr>
-      <td>layout-align-md</td>
-      <td>Sets child alignment on devices between 600px and 960px wide.</td>
-    </tr>
-    <tr>
-      <td>layout-align-gt-md</td>
-      <td>Sets child alignment on devices greater than 960px wide.
-    </tr>
-    <tr>
-      <td>layout-align-lg</td>
-      <td>Sets child alignment on devices between 960px and 1200px wide.</td>
-    </tr>
-    <tr>
-      <td>layout-align-gt-lg</td>
-      <td>Sets child alignment on devices greater than 1200px wide.</td>
-    </tr>
-  </table>
+           <td>layout-align-xs</td>
+           <td>width &lt; <b>600</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-gt-xs</td>
+           <td>width &gt;= <b>600</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-sm</td>
+           <td><b>600</b>px &lt;= width &lt; <b>960</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-gt-sm</td>
+           <td>width &gt;= <b>960</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-md</td>
+           <td><b>960</b>px &lt;= width &lt; <b>1280</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-gt-md</td>
+           <td>width &gt;= <b>1280</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-lg</td>
+           <td><b>1280</b>px &lt;= width &lt; <b>1920</b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-gt-lg</td>
+           <td>width &gt;= <b>1920/b>px</td>
+         </tr>
+         <tr>
+           <td>layout-align-xl</td>
+           <td>width &gt;= <b>1920</b>px</td>
+         </tr>
+        </table>
+
+  <br/>
+
   <p>
-   See below for more examples:
+   Below is an interactive demo that let's you explore the visual results of the different settings:
   </p>
 
   <section class="layout-panel-parent">
     <div ng-panel="layoutDemo">
-      <docs-demo demo-title='layout="{{layoutDemo.direction}}" layout-align="{{layoutAlign()}}"'
+      <docs-demo demo-title='layout="{{layoutDemo.direction}}" &nbsp; &nbsp; &nbsp; layout-align="{{layoutAlign()}}"'
```

Comment:
Not "necessary" per-se, but since this is the demo's title, it does help separate the the attributes a little bit and make them easier to read.